corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -762,61 +751,24 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
 
         self.afterRender = function () {
             self.$picker = $('#' + self.entryId);
-            var datepickerOpts = {
-                timepicker: self.timepicker,
-                datepicker: self.datepicker,
+            self.$picker.datetimepicker({
+                date: self.answer() ? self.convertServerToClientFormat(self.answer()) : self.answer(),
```

Comment:
nit: I know this is existing code, and maybe this is a weird JS thing, but is it possible to specify a value for the else case here? It's a bit strange to see `self.answer() ? self.convertServerToClientFormat(self.answer()) : self.answer()`

Reply:
do you mean instead of just using `self.answer()` I think that would be `null`, `""` or `undefined`

Reply:
Yeah so explicitly setting the date to one of those values if `self.answer()` evaluates to false. 
```suggestion
                date: self.answer() ? self.convertServerToClientFormat(self.answer()) : "",
```

Reply:
I hear you about this being weird, but I'm loath to change it without being certain what the "correct" falsy value to use is.

Reply:
Yeah my only thought is would it be easy to test using each of the falsy values locally and see which, if any break this? If that isn't straightforward don't worry about.

Reply:
yeah, that sounds not terrible, I'll give it a shot

Reply:
474e723c88b